SuperblocksHQ/ethereum-studio
# Updated Monaco editor to the 0.20.0

### Description of the Change

This PR updates the Monaco editor to the latest edition which brings some nice improvements like:

- Proper highlight language support for a ton of languages (JS / TS included) 
- Support for some new features in JS like the one described in #214 

### Verification Process

In order to make sure the updated version is working 👍 especially in regards of #214, you can simply follow these steps:

1. Create a new project from HelloWorld templates
2. Go to https://studio.ethereum.org/5e70b4e026b473001234a434 and copy the app.js into the newly created project. 
3. The following code should not highlight any errors:
```JS
// Optional Chaining is used but wrongly highlighted as an error
(window.web3?.currentProvider) ||
```

### Github Issues

This fixes #214
